runtime/vm/UpcallVMHelpers.cpp

```diff
@@ -452,7 +454,7 @@ getCurrentThread(J9UpcallMetaData *data, bool *isCurThrdAllocated)
  * @param returnStorage a pointer to the return value
  */
 static void
-convertUpcallReturnValue(J9UpcallMetaData *data, U_8 returnType, UDATA *returnStorage)
+convertUpcallReturnValue(J9UpcallMetaData *data, U_8 returnType, U_64 *returnStorage)
```

Comment:
Please fix the `#endif` comment on line 473 below.

Reply:
Fixed.